pkg/karmadactl/addons/enable.go

```diff
@@ -71,7 +72,7 @@ func NewCmdAddonsEnable(parentCommand string) *cobra.Command {
 
 	flags := cmd.PersistentFlags()
 	opts.GlobalCommandOptions.AddFlags(flags)
-	flags.IntVar(&opts.WaitPodReadyTimeout, "pod-timeout", 30, "Wait pod ready timeout.")
+	flags.IntVar(&opts.WaitComponentReadyTimeout, "component-timeout", options.WaitComponentReadyTimeout, "Wait component ready timeout.")
```

Comment:
waitPodReadyTimeout could be renamed to waitComponentReadyTimeout?
@jwcesign It also needs to be modified here, I don't know if there are other effects

Reply:
cc @lonelyCZ for checking

Reply:
It is originated from `karmadactl init`.
We can unify it .